src/pages/home_page/sub_pages/Gallery.tsx

```diff
@@ -113,20 +111,20 @@ const Gallery = (): JSX.Element => {
             }}
         >
             <ImageGrid
-                img_one={one}
-                img_two={two}
-                img_three={three}
-                img_four={four}
-                img_five={five}
-                img_six={six}
+                img_one={imgData[0]}
```

Comment:
This still isn't very extensible. Say I have 20 images in the future, then what about the remaining 8 images?
I think a better approach will be to analyze how many image data we are getting from firestore and then insert `<ImageGrid>` to the DOM dynamically using a loop.